Execute Error

```
make START_SIZE_COMBINATIONS_SZENARIO_WARNING_TAKES_HOURS
python3 -W ignore::UserWarning scenarios.size_combinations
python3: can't open file 'scenarios.size_combinations': [Errno 2] No such file or directory
Makefile:70: recipe for target 'START_SIZE_COMBINATIONS_SZENARIO_WARNING_TAKES_HOURS' failed
make: *** [START_SIZE_COMBINATIONS_SZENARIO_WARNING_TAKES_HOURS] Error 2
```

--- a/Makefile
+++ b/Makefile
@@ -67,7 +67,7 @@ $(all_len_lzma):
 
 # Start the scenario, this takes several hours and requires at least 16 GB RAM
 START_SIZE_COMBINATIONS_SZENARIO_WARNING_TAKES_HOURS:
-	$(PY) scenarios.size_combinations
+	$(PY) -m scenarios.size_combinations
 
 .PHONY: mrproper msg_size_plots plot_by_density_size_uncompressed clean_plots \
 		plot_by_density_size_zlib default \

